src/main/kotlin/org/jitsi/nlj/rtcp/RtcpRrGenerator.kt

```diff
@@ -49,11 +49,15 @@ private data class SenderInfo(
 /**
  * Retrieves statistics about incoming streams and creates RTCP RR packets.  Since RR packets are created based on
  * time (and not on a number of incoming packets received, etc.) it does not live within the packet pipelines.
+ *
+ * @param additionalPacketSupplier A function which supplies additional RTCP packets (such as REMB) to be sent together
```

Comment:
Is the RR transmission interval definitely what you want for REMBs?  I feel like once a second is kind of slow for bandwidth estimation updates.  Do we know how often Firefox sends them?

Reply:
As we discussed, we'll change the interval to 500ms